helm/slurm/values.yaml

```diff
@@ -138,7 +138,7 @@ slurm:
   # Optional raw Slurm configuration files, as a map.
   # The map key represents the config file by name; the map value represents config file contents as a string.
   # Ref: https://slurm.schedmd.com/man_index.html#configuration_files
-  configFiles: {}
+  configFiles: {gres.conf: "AutoDetect=off"}
```

Comment:
Format as YAML not JSON. Even when we're outputting the content with templating we still want it to be rendered in YAML so it's readable for debugging.

```
configFiles:
  gres.conf: "AutoDetect=off"
```